### ISSUE TITLE
Analytics FAQ section - Lifecycle hits

### ISSUE THREAD

**Issue by praveek** [AdobeDocs/aep-mobile-sdkdocs#192]
Add a FAQ item about Lifecycle metrics sent in Analytics hits. 
One common question I see is about analytics extension sending a separate Lifecycle hit sometimes, but grouping the metrics with track calls other times.

**Comment by emdobrin** [comment]
@praveek we have this information covered here: https://developer.adobe.com/client-sdks/documentation/adobe-analytics/event-reference/#lifecycle-extension
Would it help to just link this section from the FAQ page?

**Comment by praveek** [comment]
Yes. We can even link it from [Analytics FAQ](https://developer.adobe.com/client-sdks/documentation/adobe-analytics/faq/) page instead
